Sources/Basic/OutputByteStream.swift

```diff
@@ -51,73 +51,82 @@ public class OutputByteStream: TextOutputStream {
     // MARK: Data Access API
 
     /// The current offset within the output stream.
-    public var position: Int {
+    public final var position: Int {
         return buffer.count
     }
 
     /// The contents of the output stream.
     ///
     /// This method implicitly flushes the stream.
-    public var bytes: ByteString {
+    public final var bytes: ByteString {
         flush()
         return ByteString(self.buffer)
     }
     
     // MARK: Data Output API
 
-    public func flush() {
+    public final func flush() {
         // Do nothing.
     }
 
     /// Write an individual byte to the buffer.
-    public func write(_ byte: UInt8) {
+    public final func write(_ byte: UInt8) {
         buffer.append(byte)
     }
 
+    /// Write a collection of bytes to the buffer.
+    public final func write<C: Collection where C.Iterator.Element == UInt8>(collection bytes: C) {
```

Comment:
Was the parameter name required for some reason? I'd rather not have it for consistency.